Fixed missing quotes in #694

Nadahar opened the pull request:
#694 was merged with missing quotes in ```setup.nsi``` (b77f7eed07fccbf39d40cd8b73b5656bdf5030d3), this fixes that.

Nadahar commented:
@Sami32 Thanks - feel free to push any corrections to the WMP profile as well.

Sami32 commented:
And one fix haste, as you like them ;-)

I did long ago, but never withdrawed the NS... that is for Xbox or something like that i guess.
Do you have something specific in mind ?

Nadahar commented:
@Sami32 I was thinking of this:
> but i'm near sure that the WMA profiles should be updated as well:
a897a10#diff-6f9c57a374990c14f75e43d3dd15b5deR115

I'm not sure exactly what you were thinking of, but whatever it was it would be nice to fix that too in this PR.

Nadahar commented on src/main/external-resources/renderers/BlackBerry-PlayBook-KalemSoftMP.conf:
```diff
@@ -39,7 +39,7 @@ Supported = f:wmv                 m:video/x-ms-wmv
 Supported = f:mp3    m:audio/mpeg       n:2
-Supported = f:aac-lc m:audio/x-m4a      n:2   a:(?!alac).+
+Supported = f:aac    m:audio/x-m4a      n:2   a:(?!alac).+
 Supported = f:wma    m:audio/x-ms-wma
```
Ah, thanks 😊

Sami32 commented:
I cannot test it, and even try, so i leave it to users report or yourself to update this renderer.
The main point here was to fix, so feel free to merge when you want as i'm done.

Nadahar commented:
@Sami32 Yes, I didn't mean that you should "improve" it - but I understood it as if you were thinking of something with the WMA profiles that had changed since it was made. Did I misunderstand?

Sami32 commented:
Just that this one and most of the others renderers support more than the standard "wma" and that will be same for the new aac profiles. Lossless, pro, voice, wma10.
But that need users test, and not guess ;)

Nadahar commented:
The BubbleUPnP audio support was mostly wrong, so I've done some testing and corrected most of it. BubbleUPnP claims to support WMA, and they report correctly - but they never play, so I simply set all WMA to transcode.

Surprisingly their WAV support is terrible to non-existing, so I had to transcode that as well. It accepts and plays WAV files, but it's mostly clicking and noise. It seems the only formats they have implemented properly is MP3, FLAC and AAC. I didn't find anything with those formats that didn't play perfectly.

Nadahar commented:
The audio support in the WMP profile was almost as bad as BubbleUPnP, so I've corrected that as well. I haven't checked all corner cases, but my test files mostly play.

I also fixed some small things in ```FFmpegAudio``` and ```AudioUtils.parseRealAudio()``` that I found while testing.

Using the "player panel" to push is a pain because it's quite bugged, but "bump" beats all bugginess. I had to give up using "bump" because it did so many strange things. It's really a shame that this hasn't been fixed, you have to be pretty lucky to get that to work.

Sami32 commented on src/main/java/net/pms/encoders/FFmpegAudio.java:
```diff
@@ -267,6 +267,7 @@ public boolean isCompatible(DLNAResource resource) {
 			PlayerUtil.isAudio(resource, Format.Identifier.MLP) ||
 			PlayerUtil.isAudio(resource, Format.Identifier.MP3) ||
 			PlayerUtil.isAudio(resource, Format.Identifier.MPA) ||
+			PlayerUtil.isAudio(resource, Format.Identifier.MPC) ||
```
You missed to replace OGG by OGA

Sami32 commented on src/main/external-resources/renderers/Microsoft-WindowsMediaPlayer.conf:
```diff
@@ -85,51 +86,32 @@ MediaInfo = true
-Supported = f:gif   m:image/gif
-Supported = f:jpg   m:image/jpeg
-Supported = f:png   m:image/png
+Supported = f:3ga|3ga2  a:aac-lc|he-aac                     m:audio/3gpp
```
Typo ==> should be `3g2a`

ler0y replied:
keen eyes :)

Nadahar replied:
It would be nicer if these things were discovered *before* merge... 👼

Sami32 replied:
Too fast merge or too lazy reviewer ? :stuck_out_tongue_winking_eye:

ler0y commented:
Nadahar: 
>The BubbleUPnP audio support was mostly wrong, so I've done some testing and corrected most of it. BubbleUPnP claims to support WMA, and they report correctly - but they never play, so I simply set all WMA to transcode.<

The not by the bubble audio player supported audio I had given the the m:video/blabla mime type so these where forced played by the MXplayer. This worked quite fine in past but with recently bubble upnp this works quite bugged. After the audio started with the MXplayer there is no way back to the bubble upnp sever. so I Agree by deleting the m:video/blabla MXplayer audio supported lines.

WMA play fine (BubbleUPnP claims to support WMA) on my newer android phone (Samsung galaxy s6) but fails with my older phone. so I forced wma to MXplayed.

Sami32 commented:
@ler0y I don't what is the nature of your WMA files or your old phone model, but you certainly need to adjust the WMA string with the following values:
`a:wma|wmalossless`
And test with some more like:
`wma10`(will in the next version), `wmapro` or `wmavoice`

Sami32 commented on src/main/external-resources/renderers/Google-Android-BubbleUPnP-MXPlayer.conf:
```diff
@@ -96,67 +96,40 @@ UpnpDetailsSearch = BubbleUPnP Media Renderer
+Supported = f:ogg       v:divx|theora                                a:vorbis                                      m:video/ogg
+Supported = f:rm        v:rm                                         a:aac-lc|he-aac|ac3|cook|ralf                 m:application/vnd.rn-realmedia-vbr
+Supported = f:webm      v:vp8|vp9                                    a:vorbis                                      m:video/webm
+Supported = f:wmv       v:mp4|mpeg2|vc1|wmv                          a:adpcm|lpcm|mpa|wma                          m:video/x-ms-wmv
```
@ler0y It seem that WTV has been forgotten. _Someone_ will need to test it :wink: 

You already confirmed that WAV was only working with a sample rate of 48000 Hz, but others format seem to have been untested, at least without setting any MIME type and leaving UMS set the new default values automatically.
AC3, APE, WV (WavPack), MPC (MusePack), WAV, AU, OPUS (.opus ?), DSD

ler0y replied:
WTV was not been forgotten. I am almost sure that when i wrote these config files about 2 jears ago, MediaInfo could parse WTV with MPEG-2 video and MP2 audio.
Now only the WTV format is parsed.

Next Supported line should be added to the Google-Android-BubbleUPnP-MXPlayer.conf to support WTV now:
`Supported = f:wtv  m:video/wtv`

WTV test files could by found here: https://samples.mplayerhq.hu/wtv/
These test files has also DVB subtitles (not recognized by UMS ) that can be enabled in the MX Player.

Sami32 replied:
@ler0y They are planned to: https://github.com/UniversalMediaServer/UniversalMediaServer/compare/Sami32-patch-6 :wink:

Nadahar commented:
@ler0y I didn't understand your "hack" to forward audio to MXPlayer, I was sure that ```video/blabla``` was just some initial value that was pasted but never was replaced with the actual type.
